src/gui/Src/Gui/SettingsDialog.cpp

```diff
@@ -250,6 +250,7 @@ void SettingsDialog::LoadSettings()
     GetSettingBool("Gui", "ShowExitConfirmation", &settings.guiShowExitConfirmation);
     GetSettingBool("Gui", "DisableAutoComplete", &settings.guiDisableAutoComplete);
     GetSettingBool("Gui", "AsciiAddressDumpMode", &settings.guiAsciiAddressDumpMode);
+    GetSettingBool("Gui", "FollowJumpsAndCalls", &settings.guiFollowJumpsAndCalls);

```

Comment:
Just get rid of the setting, it's better to have double click be intuitive and you are not removing functionality since you can still do the actions by other means...